File: Indie Python Programming Course/step03/3.03/3.3.02.py
'''

Даны три целых числа, каждое записано в отдельной строке.

Нужно вывести значение наибольшего из данных чисел

Примечание: используйте только условный оператор, функцией max пользоваться нельзя

Sample Input 1:

5
7
21

Sample Output 1:

21

Sample Input 2:

81
75
-65

Sample Output 2:

81

Sample Input 3:

-41
7
-79

Sample Output 3:

7

Sample Input 4:

-66
-21
-78

Sample Output 4:

-21

'''

a,b,c = int(input()), int(input()), int(input())
if a > b and a > c:
    print(a)
elif b > a and b > c:
    print(b)
elif c > a and c > b:
    print(c)

File: JavaScript. A3 Tasks/3 JavaScript для начинающих/3.03/3.3.02.js
/*

Даны два целых числа k и n. Верните из функции строку, которая состоит из чисел n, повторяющихся k раз, разделенных пробелом. 
Известно, что и k и n больше либо равно 1.

Sample Input 1:

9 5

Sample Output 1:

5 5 5 5 5 5 5 5 5

Sample Input 2:

6 1

Sample Output 2:

1 1 1 1 1 1

*/

function testCycle(k, n) {
  let x = [];
  for (let i = 0; i < k; i++) {
      x.push(n)
  }
  return x.join(' ');
}

let [k, n] = prompt().split(' ').map(Number)
console.log(testCycle(k, n))
